rubin_scheduler/scheduler/example/example_scheduler.py

```diff
@@ -326,56 +332,71 @@ def blob_for_long(
 
     Parameters
     ----------
-    nside : int (DEFAULT_NSIDE)
-        The HEALpix nside to use
-    nexp : int (1)
-        The number of exposures to use in a visit.
-    exptime : float (30.)
-        The exposure time to use per visit (seconds)
-    filter1s : list of str
-        The filternames for the first set
-    filter2s : list of str
-        The filter names for the second in the pair (None if unpaired)
-    pair_time : float (33)
-        The ideal time between pairs (minutes)
-    camera_rot_limits : list of float ([-80., 80.])
+    nside : `int`
+        The HEALpix nside to use. Default to DEFAULT_NSIDE.
+    nexp : `int`
+        The number of exposures to use in a visit. Default 2.
+    exptime : `float`
+        The exposure time to use per visit (seconds).
+        Default 29.2
+    filter1s : `list` of `str`
```

Comment:
`list` [`str`]  rather than `list` of `str` (https://developer.lsst.io/python/numpydoc.html#py-docstring-parameter-types-sequences)